release-github/action.yml

```diff
@@ -173,12 +204,54 @@ runs:
       shell: bash
       run: ls -R dist/
 
+    - name: "Install toml"
+      shell: bash
+      run: |
+        python -m pip install --upgrade pip toml==${{ inputs.toml-version }}
+
+    - name: "Get CHANGELOG file location"
+      shell: python
+      run: |
+        import sys
+        sys.path.insert(1, '${{ github.action_path }}/../doc-changelog/')
+
+        from parse_pr_title import get_towncrier_config_value, save_env_variable
+
+        changelog_loc = get_towncrier_config_value("filename")
+        save_env_variable("CHANGELOG_LOC", changelog_loc)
+
+    - name: "Release to GitHub"
+      uses: softprops/action-gh-release@v1
+      env:
+        REPO_NAME: ${{ github.event.repository.name }}
+      if: inputs.dry-run == 'false' && env.CHANGELOG_LOC != ''
+      with:
```

Comment:
Can we avoid duplication of the stage using conditional inputs for the arguments and assigning defaults whenever we don't use them?